lib/service/config.js

```diff
@@ -0,0 +1,87 @@
+const debug = require('debug')('crowi:service:config')
+const uuidv4 = require('uuid/v4')
+const redis = require('redis')
+
+class Config {
```

Comment:
memo:
If this service behaves as `Config Service`, this should cover updating DB and updating config should always be done through this service instead of calling Model directly.
But anyway, at this moment, it is okay. Let's do refactor later in 1.8.